person.rb

```diff
@@ -20,6 +21,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] As requested by the previous code reviewer, which is also a requirement for this project, method for adding rentals should be present and person instant object itself should be sent as a parameter to create the new rental achieving the 'has-many' association, something like so: 

```ruby
  def add_rental(book, date)
    Rental.new(book, self, date)
  end
```

Please kindly fix that.